packages/shared/src/components/history/ReadingHistoryItem.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { HideReadHistoryProps, ReadHistory } from '../../graphql/users';
+import XIcon from '../../../icons/x.svg';
+import classed from '../../lib/classed';
+import { Button } from '../buttons/Button';
+import { LazyImage } from '../LazyImage';
+
+interface ReadingHistoryItemProps {
+  className?: string;
+  history: ReadHistory;
+  onHide?: (params: HideReadHistoryProps) => Promise<unknown>;
+}
+
+const SourceShadow = classed(
+  'div',
+  'absolute left-5 -my-1 w-8 h-8 rounded-full bg-theme-bg-primary',
+);
+
+function ReadingHistoryItem({
+  history: { timestamp, post },
+  onHide,
+  className,
+}: ReadingHistoryItemProps): ReactElement {
+  return (
+    <section className="flex relative items-center hover:bg-theme-hover">
```

Comment:
Why do we choose section here?

My initial thoughts would be it would look like a list? (ul > li)
Maybe we could then do (section > article) as these are articles

Reply:
Totally makes sense. Let's go for the `section` for the root container of the list and `article` for each item.